### PR TITLE
Add reporting node balance to hactar backend

### PR DESCRIPTION
**Pull request by MakMuftic**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -1,9 +1,11 @@
 stats:
-  interval: 10 #seconds
+  interval: 60 #seconds
   blocks:
-    interval: 50 #seconds
+    interval: 10 #seconds
```
Comment (morrigan) [inline config.yaml]: not sure why you updated but average block time is around 50 seconds

File: config.yaml [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
   uptime:
-    interval: 5 #seconds
+    interval: 10 #seconds
+  balance:
```
Comment (morrigan) [inline config.yaml:7]: Maybe for balance we can put amount 2 or 3 times higher than blocktime

File: internal/lotus/wallet.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	GetWalletDefaultAddress() (string, error)
+	GetWalletBalance(walletAddress string) (string, error)
+}
+
+type walletService struct {
+	client *Client
+}
+
+func (ws *walletService) GetWalletDefaultAddress() (string, error) {
+	response, err := ws.client.lotusNodeClient.Call(lotus.WalletDefaultAddress)
```
Comment (morrigan) [inline internal/lotus/wallet.go:18]: Not related to PR but... Are we reporting this address somewhere? I think it should be in general miner info (as it is in design there) but we kinda forgot about it.

**Review by morrigan — CHANGES_REQUESTED** [review]
Looks good! Let's just update the intervals in this PR

File: internal/lotus/wallet.go (hunk 3 above)
Comment (MakMuftic) [reply internal/lotus/wallet.go:18]: I will create another task and PR for this :+1:

**Review by morrigan — APPROVED** [review]
Looks good!

**Pull request merged**